.github/workflows/hypertrace-ingester/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: "2.4"
+services:
+
+  hypertrace:
+    image: hypertrace/hypertrace:main
+    container_name: hypertrace
+    environment:
+      - MONGO_HOST=mongo
+      - ZK_CONNECT_STR=zookeeper:2181/hypertrace-views
+    ports:
+      - 2020:2020
+    healthcheck:
+      start_period: 20s
+    depends_on:
+      mongo:
+        condition: service_healthy
+      kafka-zookeeper:
+        condition: service_healthy
+      pinot:
+        condition: service_started
+
+  hypertrace-collector:
+    image: hypertrace/hypertrace-oc-collector
+    container_name: hypertrace-collector
+    command: ["--config=/config/collector-config.yaml"]
+    ports:
+      - 14268:14268 # Jaeger Thrift
+      - 14267:14267 # Jaeger HTTP
+      - 9411:9411 # Zipkin HTTP
+    networks:
+      default:
+        aliases:
+          - jaeger
+          - jaeger-collector
+          - zipkin
+    depends_on:
+      kafka-zookeeper:
+        condition: service_healthy
+
+  hypertrace-ingester:
+    image: hypertrace/hypertrace-ingester:test
```

Comment:
sut should be this container. We shouldn't forward reference containers that depend on us

Reply:
done